File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md

```diff
@@ -452,6 +452,45 @@ The runtime files, or blink binaries, will be copied into a bin or published fol
 
 By including the <b><ExcludeAssets>native</ExcludeAssets></b> option in the package reference of the csproj file, you can exclude the runtime files or blink binaries from being copied into the bin or publish folder while building and publishing the application. But you need to place the BlinkBinaries in the server disk and set the BlinkPath in the BlinkConverterSettings to perform the conversion. 
 
+## Preservation issues occurs while converting HTML(HTTPS sites) to PDF using WebKit rendering engine in local machine/Azure App service
+
+<table>
+<th style="font-size:14px" width="100px">Issue
+</th>
+<th style="font-size:14px">Preservation issues occurs while converting HTML(HTTPS sites) to PDF using WebKit rendering engine in local machine/Azure App Service. 
+</th>
+
+<tr>
+<th style="font-size:14px" width="100px">Reason
+</th>
+<td>The issue occurs due to missing of OPENSSL assemblies in your machine.
+</td>
+</tr>
+
+<tr>
+<th style="font-size:14px" width="100px">Solution
+</th>
+<td>
+Our WebKit rendering engine requires OPENSSL libraries to be installed in the machine where the conversion takes place. 
```

Comment:
Our WebKit rendering engine requires OPENSSL assemblies for accessing the resources from HTTPS URL. If you are accessing any resources from HTTPS sites, then kindly include these below assemblies in your project,
<img src="htmlconversion_images/OPENSSl_assemblies.png" alt="ExcludeAssets"><br><br>

You can get the OPENSSL assemblies from the below link, 
<a href="https://www.syncfusion.com/downloads/support/directtrac/general/ze/OPENSSL-798051511">https://www.syncfusion.com/downloads/support/directtrac/general/ze/OPENSSL-798051511</a>